apps/heureka/src/components/ServiceDetails/ImageVersionDetailsPanel/ImageVersionIssuesList/index.tsx

```diff
@@ -47,8 +51,25 @@ export const ImageVersionIssuesList = ({ serviceCcrn, imageVersion }: ImageVersi
     }
   }, [error])
 
+  const handleSearch = (term: string) => {
+    setSearchTerm(term)
+  }
+
+  const handleClear = () => {
+    setSearchTerm("")
+  }
+
   return (
     <>
+      <Stack gap="2" className="mb-4 mt-8">
+        <ContentHeading>Issues</ContentHeading>
+        <SearchInput
+          placeholder="Search for CVE number"
+          className="w-96 ml-auto"
+          onSearch={handleSearch}
```

Comment:
since the function just does `setSearchTerm(term)` we could do it inline and save two extra methods. What do you think?